### DISCUSSION TITLE
UPDATE SUGGESTIONS!

### DISCUSSION THREAD

**Discussion by Jackhammer9** [JackhammerYT/RedDownloader#19] [Ideas]
### Do you have a feature request or suggestion?

Drop down all your feature requests and ideas and as the updates come by I'll make sure to add them. This thread is meant to be a collective contribution towards enriching this package. All the update ideas can be posted here.

**Comment by Jackhammer9** [comment]
[Feature]

- Add options like "today", "this week", "this moth" while sorting by hot

**Comment by LoganAC34** [comment]
- It would be nice to show an example url in the readme of the repository. I had an issue that posted about here:
https://github.com/JackhammerYT/RedDownloader/issues/27 
and then figured out later after digging in the source code that I was missing part of the url I didn't know was important. 
- In addition, it would be nice if we didn't need to include "http://www.reddit.com" in the url, and instead only need to include the "/r/..." part. That way, it natively works with PRAW using submission.permalink.
